packages/dapp/src/App.tsx

```diff
@@ -1,9 +1,7 @@
+import { Menus, Navbar, Widgets } from 'components';
```

Comment:
This works just fine in dev mode, yarn build and all - but when committing it throws me an error on all imports I have modified:

`packages/dapp/src/App.tsx(1,40): error TS2307: Cannot find module 'components' or its corresponding type declarations.`

Reply:
This commit seems to have fixed it!

https://github.com/lifinance/jumper.exchange/pull/658/commits/140021105350458d6262b93ccdc964e938616737